_inc/client/components/data/query-assistant-data/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { fetchRecommendationsData, isFetchingRecommendationsData } from 'state/recommendations';
+import { isOfflineMode } from 'state/connection';
+
+class QueryRecommendationsData extends Component {
```

Comment:
This file looks identical to `query-recommendations-data/index.jsx`. Is it expected?

Reply:
I missed this one during a refactor, I've deleted it now.